pom.xml

```diff
@@ -1518,36 +1525,6 @@
         <artifactId>maven-enforcer-plugin</artifactId>
       </plugin>
       <plugin>
-        <groupId>org.codehaus.mojo</groupId>
```

Comment:
Cool, I imagine we'd know if it failed to comiple anything, so that would verify it. Let me get an initial test going